### ISSUE TITLE
Link elements in TC2 MAPs are not optimised for a11y or mobile

### ISSUE THREAD

**Issue by karinathomasbbc** [bbc/simorgh#5539]
**Describe the bug**
`Link` elements from Legacy TC2 MAPs do not contain alt-text and a url for enhancedmobile, unlike modern MAPs

**To Reproduce**
1. View the AReS response for a legacy MAP e.g. https://ares-broker.api.bbci.co.uk/api/asset/mundo/video_fotos/2016/05/160502_video_apnea_record_rubridge_nezolandes_amv
2. Search for '<link' in the response
3. Note that the link element is as follows:

`<link type=\"page\"><caption>\n\npágina de Facebook de Vertical Club</caption><url href=\"https://www.facebook.com/verticalblue/?fref=ts\" platform=\"highweb\"/></link>`

**Expected behaviour**
1. View the AReS response for a modern (CPS) MAP with a link e.g. https://ares-broker.test.api.bbci.co.uk/api/asset/pidgin/23248703
2. Search for '<link' in the response
3. Note that the link element is as follows:
`<link><caption>this is the link text</caption><altText>this is the alt text </altText><url href=\"https://bbc.com/pidgin\" platform=\"highweb\"/><url href=\"https://bbc.com/pidgin\" platform=\"enhancedmobile\"/></link>`

- [ ] Check whether this is an issue with Content Store

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by RichardPK** [comment]
We spotted that the `<altText>` appears to be its own ‘component’ coming back from Ares, rather than an attribute: `<link><caption>this is the link text</caption><altText>this is the alt text </altText><url href=\"https://bbc.com/pidgin\" platform=\"highweb\"/><url href=\"https://bbc.com/pidgin\" platform=\"enhancedmobile\"/></link>` but after a quick look I can’t see if/how that’s being rendered in Simorgh. Neither within the code, or in the html that makes it into the browser. 

![Screenshot 2020-02-19 at 11 00 17](https://user-images.githubusercontent.com/9042672/74828447-0b703180-5307-11ea-8140-c2b90372c077.png)

**Comment by karinathomasbbc** [comment]
Look in the psammead repo: `packages/utilities/psammead-rich-text-transforms/src/candy-xml.js`

**Comment by gavinspence** [comment]
In order for a link to be accessible it either needs to be in plain text, or for an image, alt text. Given our links are plain text this should be fine. Closing as we won't change.
